packages/react-hooks/src/useDebouncedValue.test.ts

```diff
@@ -0,0 +1,65 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import useDebouncedValue, { DEFAULT_DEBOUNCE_MS } from './useDebouncedValue';
+
+beforeEach(() => {
+  jest.useFakeTimers();
+});
+
+afterAll(() => {
+  jest.useRealTimers();
+});
+
+it('should return the initial value', () => {
+  const value = 'mock value';
+  const { result } = renderHook(() => useDebouncedValue(value));
+  expect(result.current).toBe(value);
+});
+
+it('should return the initial value after the debounce time has elapsed', () => {
+  const value = 'mock value';
+  const { result, rerender } = renderHook(() => useDebouncedValue(value));
+  expect(result.current).toBe(value);
+  rerender();
+  act(() => {
+    jest.advanceTimersByTime(DEFAULT_DEBOUNCE_MS);
+  });
+  expect(result.current).toBe(value);
```

Comment:
Also `expect(result.all.length).toBe(1)` I think?

Reply:
No, it should be 2 here - 1 for the initial render, 1 for the rerender. But yea I'll add the check, good call.